### ISSUE TITLE
Batchalign pipeline: errors of transcribing MP3s to Mandarin

### ISSUE THREAD

**Issue by Wanlin-z** [TalkBank/batchalign2#14]
Hi, I am working with monolingual Mandarin-speaking children and would love to use Batchalign to analyze their language samples. When I want to transcribe the audio file with the help of Whisper, I get an error and the analysis cannot go forward. The error is:  _extract_token_timestamps() got an unexpected keyword argument 'num_input_ids'. Here is my script below:

Successfully installed batchalign-0.7.4 tokenizers-0.21.0 transformers-4.47.1

C:\Users\lenovo>py -m batchalign transcribe --lang=zho --whisper "C:\ba_input" "C:\ba_output"

Mode: transcribe; got 1 transcript to process from C:\ba_input:

Device set to use cpu
You have passed task=transcribe, but also have set `forced_decoder_ids` to [[1, None], [2, 50360]] which creates a
conflict. `forced_decoder_ids` will be ignored in favor of task=transcribe.
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a
consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
WhisperModel is using WhisperSdpaAttention, but `torch.nn.functional.scaled_dot_product_attention` does not support
`output_attentions=True` or `layer_head_mask` not None. Falling back to the manual attention implementation, but
specifying the manual implementation will be required from Transformers version v5.0.0 onwards. This warning can be
removed using the argument `attn_implementation="eager"` when loading the model.
From v4.47 onwards, when a model cache is to be returned, `generate` will return a `Cache` instance instead by default
(as opposed to the legacy tuple of tuples format). If you want to keep returning the legacy format, please set
`return_legacy_cache=True`.
  CH1M.mp3 ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━   0% 0:00:27 FAIL

ERROR on file CH1M.mp3: _extract_token_timestamps() got an unexpected keyword argument 'num_input_ids'

Thank you very much for your work, which enables us to save time with this program. I look forward to your response.
